code/modules/research/designs/weapon_designs.dm

```diff
@@ -52,13 +62,156 @@
 	desc = "Designed to quickly reload revolvers. Rubber bullets are bouncy and less-than-lethal."
 	id = "c38_rubber"
 	build_type = PROTOLATHE | AWAY_LATHE
-	materials = list(/datum/material/iron = SHEET_MATERIAL_AMOUNT * 10)
+	materials = list(/datum/material/iron = HALF_SHEET_MATERIAL_AMOUNT * 10)
 	build_path = /obj/item/ammo_box/c38/match/bouncy
 	category = list(
 		RND_CATEGORY_WEAPONS + RND_SUBCATEGORY_WEAPONS_AMMO
 	)
 	departmental_flags = DEPARTMENT_BITFLAG_SECURITY
 
+/datum/design/c38_phasic
+	name = "Speedloader (.38 Phasic) (Lethal)"
+	desc = "Designed to quickly reload revolvers. These bullets are made from an experimental alloy, 'Ghost Lead', that allows it to pass through almost any non-organic material."
+	id = "c38_phasic"
+	build_type = PROTOLATHE | AWAY_LATHE
+	materials = list(
+		/datum/material/iron = HALF_SHEET_MATERIAL_AMOUNT * 10,
+		/datum/material/bluespace = HALF_SHEET_MATERIAL_AMOUNT,
+	)
+	build_path = /obj/item/ammo_box/magazine/m38/phasic
+	category = list(
+		RND_CATEGORY_WEAPONS + RND_SUBCATEGORY_WEAPONS_AMMO
+	)
+	departmental_flags = DEPARTMENT_BITFLAG_SECURITY
```

Comment:
I still stand against phasic rounds being available to the crew, even if they're at the very end of the techweb. Full AP with high damage is bollocks

Reply:
For the record, 5 shots to crit anybody is less than 3 bursts (20 damage per shot) which is pretty strong compared to lasers which have 25 damage with 0 AP ***and*** need recharging (which makes them inherently weaker than mag-based weapons)

Reply:
Maybe locked behind illegal tech, like someone suggested?

Reply:
resolved this by removing phasic and making true strike bounce a lot, and accurately, but reducing the damage by a lot (15).